.github/workflows/promote_branch.yaml

```diff
@@ -24,6 +24,12 @@ on:
         type: boolean
         required: true
         default: false
+      force-hotfix:
+        description: |
+          force-hotfix: Allow force push when hotfixes are detected.Hotfixes will be added to source branch
+        type: boolean
+        required: true
+        default: true
```

Comment:
Do we want this behavior to be true by default?

Reply:
I think so. I can't think of when we wouldn't want the hot fix process to happen